func/functions_example_problems.py

```diff
@@ -0,0 +1,102 @@
+########################## Example Problems ##########################
+
+# MAKES TWENTY: Given two integers, return True if the sum of the integers is 20 or if one of the integers is 20.
+# If not, return False
+
+def makes_twenty(num1, num2):
+    if (num1 + num2 == 20) or num1 == 20 or num2 == 20:
+        return True
+    else:
+        return False
+
+# ANIMAL CRACKERS: Write a function takes a two-word string and returns True if both words begin with same letter
+
+# animal_crackers('Levelheaded Llama') --> True
+# animal_crackers('Crazy Kangaroo') --> False
+
+def animal_crackers(string):
+    for idx, letter in enumerate(string):
+        if letter == " ":
+            if string[idx + 1] == string[0]:
+                return True
+            else:
+                return False
+
+
+# MASTER YODA: Given a sentence, return a sentence with the words reversed
+
+# master_yoda('I am home') --> 'home am I'
+# master_yoda('We are ready') --> 'ready are We'
+
+def master_yoda(string):
+   string_list = string.split(" ")
+   #string_list = ["I", "am", "home"]
+   final_string = ""
+   for index in range(len(string_list)-1, -1, -1):
+        final_string = final_string + string_list[index] + " "
+        print(final_string) 
+   print(final_string)     
+
+
+# PAPER DOLL: Given a string, return a string where for every character in the original there are three characters
+def paper_doll(string):
+    word = ""
+    for letter in string:
+        word = word + letter + letter + letter
+    print(word)        
+# paper_doll('Hello') --> 'HHHeeellllllooo'
+# paper_doll('Mississippi') --> 'MMMiiissssssiiippppppiii'
+
+
+# BLACKJACK: Given three integers between 1 and 11, if their sum is less than or equal to 21, return their sum.
+# If their sum exceeds 21 and there's an eleven, reduce the total sum by 10.
+# Finally, if the sum (even after adjustment) exceeds 21, return 'BUST'.
+# NOTE: You must use a loop in your function, and you are not allowed to check if a number is 11 just
+# by doing number == 11 :)
+
+def blackjack(num1, num2, num3):
+    list = [num1, num2, num3]
+    print(list)
+    sum = num1 + num2 + num3
+    
+    for num in list:
+        if sum <= 21:
+            return sum
+     
+        
+        if 11 in list:
```

Comment:
Don't make any change to this, but how else can you check if the current number in your loop `num` is 11? Any thoughts?

Reply:
@fiifikrampah  Besides using a conditional?

Reply:
You have to use a conditional, right? What other conditional statement can you use to check?

Reply:
Would it be:
if num == 11: ?

Reply:
Besides that...

Reply:
while num == 11 ?

Reply:
No that's not what you want, is it?

Reply:
if num % 11 == 0: ?
         or
if num/11 == 1: ?

Reply:
What do those statements tell us? Is that what we want?

Reply:
I believe we're checking if there is an eleven in the list and if num % 11 == 0, tells that number is divisible by 11 without a reminder and hence the num is 11 and also the second one divides the num by 11 and the answer == 1 shows the number is 11, so aren't those other ways of checking that condition?

Reply:
Right,so with the `num % 11==0` piece, is that sufficient to tell us that the number is 11? With the `num / 11 == 1`, is that check by itself enough? Or are you thinking of combining the two, and is that what we want?

Reply:
Lol, I think the first one num % 11 == 0 piece is not sufficient cos I guess other numbers can divide without reminders so scratch that lol but I think the second one is valid cos only 11 can divide itself with product being one, so for that I think it's enough?

Reply:
Lol right on the second part…but if we still wanted to use the first logic what will we need in addition to that modulus logic?

Reply:
We can combine it by doing num % 11 == 0 and num/11 == 1 to check that but that looks too complicated cos just the second part can check without the first part

Reply:
Right, so what other check can we combine with that one?

Reply:
Well considering that we only dealing with numbers from 1 to 11, you could also say if num > 10, which will be eleven?